src/sentry/static/sentry/app/components/events/eventRelatedEvents.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import {Location} from 'history';
+import uniqBy from 'lodash/uniqBy';
+import moment from 'moment-timezone';
+
+import {t} from 'app/locale';
+import {Organization, Event} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import DiscoverQuery from 'app/utils/discover/discoverQuery';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import EventDataSection from 'app/components/events/eventDataSection';
+import {ALL_ACCESS_PROJECTS} from 'app/constants/globalSelectionHeader';
+import {getTraceDateTimeRange} from 'app/components/events/interfaces/spans/utils';
+
+import RelatedEvents from './relatedEvents';
+import EmptyState from './relatedEvents/emptyState';
+import {getCurrentLocation} from './relatedEvents/utils';
+import getDiscoverButton from './relatedEvents/getDiscoverButton';
+
+type Props = {
+  location: Location;
+  event: Event;
+  organization: Organization;
+};
+
+type State = {
+  isLoading: boolean;
+  orgFeatures: Set<string>;
+  orgSlug: string;
+  eventView?: EventView;
+};
+
+class EventRelatedEvents extends React.Component<Props, State> {
+  state: State = {
+    isLoading: true,
+    orgFeatures: new Set(this.props.organization.slug),
```

Comment:
```suggestion
    orgFeatures: new Set(this.props.organization.features),
```

Reply:
Why put org features in state? Same with orgslug? Arn't both of those just derived properties?

Reply:
good catch!!! 👏

Reply:
@EvanPurkhiser yes you are right, but I didn't want to use "new Set(...)" in every re-render..that is the reason 😉